easybuild/easyconfigs/a/Autoconf/config.guess_ppc64le.patch

```diff
@@ -0,0 +1,645 @@
+--- a/build-aux/config.guess	2012-04-25 04:15:10.000000000 +0200
```

Comment:
@omula Please include a short description + URL for upstream source of this patch above this line